test/test_bdv_datasets.py

```diff
@@ -0,0 +1,183 @@
+import os
+import unittest
+from abc import ABC
+from shutil import rmtree
+
+import numpy as np
+from pybdv import make_bdv, BdvDataset
+from pybdv.util import open_file, get_key, HDF5_EXTENSIONS
+
+try:
+    import z5py
+except ImportError:
+    z5py = None
+
+DOWNSCALE_MODE = 'interpolate'
```

Comment:
I would suggest using `mean` as the default down-scaling mode. `interpolate` is slower and (usually) yields worse results. I would also set this as default in the `BdvDataset` class (sorry I don't see this anymore, it's hidden i old changes).

Reply:
Done